Translators/RoadConditions/itis_codes.py

```diff
@@ -0,0 +1,74 @@
+from enum import Enum
+
+
+class ItisCodes(Enum):
```

Comment:
Is there a way to share this with the other translators?

Reply:
Yes, I'm planning on pulling shared methods & classes out to a shared library

Reply:
In an upcoming PR I assume then?

Reply:
Yes, I've created a work item for this and will link the relevant PR here when it's up

Reply:
The PR to add a Shared package can be viewed here: #24